pkgs/tools/networking/wait-for-it/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "wait-for-it-unstable";
+  version = "unstable-2020-02-05";
+
+  src = fetchFromGitHub {
+    owner = "vishnubob";
+    repo = "wait-for-it";
+    rev = "c096cface5fbd9f2d6b037391dfecae6fde1362e";
+    sha256 = "1k1il4bk8l2jmfrrcklznc8nbm69qrjgxm20l02k01vhv2m2jc85";
+  };
+  installPhase = ''
```

Comment:
While I'm not sure about the exact policy on this in nixpkgs, but wouldn't `buildCommand` fit better here?

Reply:
As above, I just went off examples I found around, not a specific guide or documentation. Maybe buildCommand is simpler? I can't find any specific documentation around it.

Reply:
With the current expression, there are multiple implicit phases apart from `installPhase` that are run. First is the expected `unpackPhase` that copies the source to `$NIX_BUILD_TOP`, then comes the `configurePhase` which tries to run `,/configure` (which doesn't make sense here), then the `buildPhase` which tries to run `make` (which again, doesn't make sense here), then `installPhase` and then `fixupPhase` that fixes shebangs in scripts. 
Now that I think about it, leaving it as-is and adding `phases = [ "unpackPhase" "installPhase" "fixupPhase" ];` makes more sense.

Reply:
Also done. Thanks. As always, curious how to help the next contributor just know to do this, rather than consuming your time.

Reply:
Though note that with explicit phases, you exclude those not listed, for example `patchPhase`, so adding `patches` field will not work. See also https://github.com/NixOS/nixpkgs/issues/28910